### PR TITLE
[Homework #20] - Routing-redux by Nicolás Roberto

### PR DESCRIPTION
**Pull request by nicolasroberto**

### CODE REVIEW

File: homeworks/nicolas-roberto/20-routing-redux/react-router-tutorial/package.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "preview": "vite preview"
+  },
+  "dependencies": {
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0"
```
Comment (solvd-mmelnyk) [inline homeworks/nicolas-roberto/20-routing-redux/react-router-tutorial/package.json:13]: I see that you use next packages in project:
1. react-router-dom
2. localforage
3. match-sorter
4. sort-by
but I don't see them in `package.json` file

Could you please double-check?

File: homeworks/nicolas-roberto/20-routing-redux/redux-toolkit/src/App.jsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
+import React, { useState } from 'react';
+import './App.css';
+import TodoForm from './components/TodoForm';
+import TodoList from './components/TodoList';
+
+export default function App() {
+  /* Single todo item (initial value is an empty string) */
+  const [todo, setTodo] = useState('');
```
Comment (solvd-mmelnyk) [inline homeworks/nicolas-roberto/20-routing-redux/redux-toolkit/src/App.jsx:8]: You used just state
I don't see using Redux Toolkit to handle state.
https://redux-toolkit.js.org/

**Review by solvd-mmelnyk — CHANGES_REQUESTED** [review]
Please check my notes!

File: homeworks/nicolas-roberto/20-routing-redux/react-router-tutorial/package.json (hunk 1 above)
Comment (nicolasroberto) [reply homeworks/nicolas-roberto/20-routing-redux/react-router-tutorial/package.json]: Thanks, i think i fixed it!

**Review by solvd-mmelnyk — APPROVED** [review]
Good Work!

**Pull request merged**